JS-Algorithms/challenges.js

```diff
@@ -13,13 +13,30 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let minutes = 0;
+  let hours = 0;
+
+  minutes = Math.floor(seconds / 60); //Division always returns a float number, so I need to round it down
+  seconds = seconds % 60; // The residual from dividing the seconds
+
+  if (minutes >= 60) {
+    //Applying the same logic if there are over 60 minutes,
+    hours = Math.floor(minutes / 60);
+    minutes = minutes % 60;
+  }
+
+  //Manually formatting string, adding 0 if necessary
+  result = `${hours > 9 ? hours : `0${hours}`}:${
```

Comment:
we need to initialize the constant, if possible, return this string directly in the return sentence in L33

```suggestion
  const result = `${hours > 9 ? hours : `0${hours}`}:${
```